### ISSUE TITLE
Vertex Colors usage missing in the spec

### ISSUE THREAD

**Issue by migueldgg** [KhronosGroup/glTF#1998]
The glTF 2.0 spec used to state the following:

> In addition to the material properties, if a primitive specifies a vertex color using the attribute semantic property `COLOR_0`, then this value acts as an additional linear multiplier to `baseColor`.

This sentence was removed in commit d6e9c90, making it unclear how vertex colors should be used in the final color computation.

It would be important to get this crucial bit of information back in the specification.

**Comment by mlimper** [comment]
TBH, it's not perfectly clear to me yet how vertex colors are assumed to be stored / used... both questions are connected.

storage: sRGB vs. linear?
use case: multiply brightness / ao vs. storing actual colors?

If someone has some insight into what was originally intended, that would probably be helpful in order to add more details / explanation to the spec

**Comment by lexaknyazev** [comment]
It's restored in the new spec draft (#1997).

**TL;DR**
- Base color is given by a `baseColorTexture`, that is 8-bit sRGB.
- When the texture slot is empty, the texture is defined as returning `(1.0, 1.0, 1.0, 1.0)` for all pixels.
- `baseColorFactor` and `COLOR_0` values are linear multipliers.

**Comment by lexaknyazev** [comment]
The updated spec has been released.
